apps/felvin-faq/src/App.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import styled from "styled-components";
+import { isTriggered } from "@felvin-search/core";
+import fingerprint from 'talisman/keyers/fingerprint';
+import sift4 from 'talisman/metrics/sift4';
+
+// ------------- Preprocessors ---------------------
+// Move to @felvin-search/core
+
+// fingerprint
+
+// ------------- String Matchers -------------------
+const identityMatcher = (string1, string2) => {
+  return string1 === string2;
+}
+
+const siftMatcher = (string1, string2) => {
+  const N = Math.max(string1.length, string2.length)
+  return sift4(string1, string2)/(N*1.0)
+}
+
+// Default matcher
+const matchStrings = (string1, string2, threshold=0.8, preprocessor=fingerprint, matcher=identityMatcher) => {
```

Comment:
@N-Shar-ma @sahil-shubham Here's my attempt at creating smarter matchers. See if these are working for you locally.